src/components/application_manager/include/application_manager/application_data_impl.h

```diff
@@ -34,6 +34,8 @@
 #define SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_APPLICATION_DATA_IMPL_H_
 
 #include <string>
+#include <map>
+#include <cstdint>
```

Comment:
@IGapchuk add extra space between C++ libs and project libs

Reply:
@AKalinich-Luxoft Added in fbc582d9bf3feaa96a345733d977106eea7a02b4.